pkg/declextract/entity.go

```diff
@@ -89,6 +89,13 @@ type FileOps struct {
 	Mmap       string `json:"mmap,omitempty"`
 	Ioctl      string `json:"ioctl,omitempty"`
 	SourceFile string `json:"source_file,omitempty"`
+
+	open  *Function
```

Comment:
So these are filled in in `resolveFopsCallbacks()` on the basis of the corresponding capitalized fields, right?

Looks somewhat obscure. Maybe it could be some separate data structure? Or at least we need some more comments here.

Reply:
Moved to a separate struct. Now code will panic if it's used before it was inited.